CRM/Utils/Check/Component/Env.php

```diff
@@ -688,6 +688,51 @@ public function checkExtensions() {
     return $messages;
   }
 
+  /**
+   * @return CRM_Utils_Check_Message[]
+   */
+  public function checkScheduledJobLogErrors() {
+    $jobs = civicrm_api3('Job', 'get', [
+      'sequential' => 1,
+      'return' => ["id", "name", "last_run"],
+      'is_active' => 1,
+      'options' => ['limit' => 0],
+    ]);
+    $html = '';
+    foreach ($jobs['values'] as $job) {
+      $lastExecutionMessage = civicrm_api3('JobLog', 'get', [
+        'sequential' => 1,
+        'return' => ["description"],
+        'job_id' => $job['id'],
+        'options' => ['sort' => "id desc", 'limit' => 1],
+      ])['values'][0]['description'] ?? NULL;
+      if (!empty($lastExecutionMessage) && strpos($lastExecutionMessage, 'Failure') !== FALSE) {
+        $viewLogURL = CRM_Utils_System::url('civicrm/admin/joblog', "jid={$job['id']}&reset=1");
+        $html .= "<tr><td>{$job['name']} </td><td>{$lastExecutionMessage}</td><td>{$job['last_run']}</td><td> <a href='{$viewLogURL}'>View Job Log</a></td></tr>";
+      }
+    }
+    if (empty($html)) {
+      return [];
+    }
+
+    $message = "<p>The following scheduled jobs failed on the last run:</p>
```

Comment:
Could you wrap the strings in `ts()` ? (this line, and other lines around it)

Reply:
Sorry for the late update @mlutfy. I've added the ts() now. Thanks.